app/controllers/AddAdditionalMetadataController.scala

```diff
@@ -3,7 +3,7 @@ package controllers
 import auth.TokenSecurity
 import configuration.{GraphQLConfiguration, KeycloakConfiguration}
 import controllers.AddAdditionalMetadataController.{File, formFieldOverrides}
-import controllers.util.MetadataProperty.{clientSideOriginalFilepath, description, descriptionClosed}
+import controllers.util.MetadataProperty.{clientSideFileLastModifiedDate, clientSideOriginalFilepath, closurePeriod, closureStartDate, description, descriptionClosed, end_date}
```

Comment:
I'd shorten this import statement to: `import controllers.util.MetadataProperty._`